contracts/StablecoinConverter.sol

```diff
@@ -2,13 +2,14 @@ pragma solidity ^0.5.0;
 
 import "./EpochTokenLocker.sol";
 import "./libraries/IdToAddressBiMap.sol";
+import "@gnosis.pm/solidity-data-structures/contracts/libraries/IterableAppendOnlySet.sol";
 import "solidity-bytes-utils/contracts/BytesLib.sol";
-import "./libraries/IdToAddressBiMap.sol";
```

Comment:
I can see that this dependency was removed, but I can't see what it was ever used for. Am I perhaps looking at only a subset of changes?

Reply:
It's used for the mapping of tokenId to ERC20 addresses, but was imported twice, so I removed one of the imports.

Reply:
Ahh yes I had noticed that on a previous PR of @josojo's

Reply:
If so, I am sorry Ben